commands/fetch_templates.go

```diff
@@ -51,12 +52,20 @@ func fetchTemplates() error {
 		}
 	}
 
+	log.Printf("Cleaning up zip file...")
+	if _, err := os.Stat(zipFileName); err == nil {
+		os.Remove(zipFileName)
+	} else {
+		return err
+	}
+
 	return err
 }
 
 func fetchMasterZip() error {
 	var err error
 	if _, err = os.Stat("master.zip"); err != nil {
+		err = nil
```

Comment:
That `err` variable is being set to `nil` because after we find that there is an error, we load the master.zip file location from the `templateUrl` environment variable, or if that env variable does not exist, we grab it from github. The error variable is not needed here, in fact it is a bug if it is not reset to nil, because if you have `templateURL` env variable set because you intend to load the master.zip from an alternative location, as this test does, then the test will normally fail on "master.zip os.Stat method failed", because that method and set the `err` at line 67. Let me know if there is a better way. Thanks